src/scripts/main.js

```diff
@@ -1,3 +1,21 @@
 'use strict';
 
-// write code here
+const list = document.querySelector('ul');
+
+list.querySelectorAll('li').forEach((el) => {
```

Comment:
Consider checking if `el.firstChild` is not null before accessing its `textContent`. This will prevent potential errors if a list item does not have a child node.